src/db.py

```diff
@@ -98,7 +102,9 @@ def remove_query(self, job: JobId) -> None:
         """Sets the job status to removed."""
         with self.session() as session:
             session.execute(
-                update(Job).where(Job.id == job).values(status="removed")
+                update(Job)
+                .where(Job.id == job)
+                .values(status=JobStatus.cancelled)
```

Comment:
Again, the idea of #414 is to drop the jobs from the database. So this should be delete instead of update.